addOns/reports/src/main/zapHomeFiles/reports/traditional-html/report.html

```diff
@@ -292,6 +395,92 @@ <h3 th:text="#{report.alerts.detail}">Alert Detail</h3>
 			<div class="spacer"></div>
 		</th:block>
 	</th:block>
+
+	<th:block th:if="${reportData.isIncludeSection('sequencedetails')}">
+		<th:block th:if="${reportData.isIncludeSection('sequencedetails')}">
+			<th:block
+				th:if="${reportData.reportObjects.get('seqAScanData') != null}">
+				<h3 th:text="#{report.sequences.details.name}">Sequences
+					Details</h3>
+				<sup>With the associated active scan results.</sup>
+
+				<th:block
+					th:each="seq, seqState: ${reportData.reportObjects.get('seqAScanData').seqData}">
+					<h4 th:text="${seq.sequenceName}">sequenceName</h4>
+					<div class="lm2">
+						<th:block th:each="step, stepState: ${seq.getSteps()}">
+
+							<h4>
+								<th:block
+									th:text="#{report.sequences.step.label} + ' ' + ${stepState.index + 1} + ': ' + ${step.getOriginalMsg().getRequestHeader().getMethod()} + ' - '">sequence heading</th:block>
+								<a
+									th:href="${step.getOriginalMsg().getRequestHeader().getURI().toString()}"
+									th:text="${step.getOriginalMsg().getRequestHeader().getURI().toString()}">URL</a>
+							</h4>
+
+							<div class="lm2">
+
+								<span th:text="#{report.sequences.step.result}">Result</span> <span
+									th:text="${step.getResult()}">result</span><br> <span
+									th:if="${step.getAlerts().size() &gt; 0}"
+									th:text="#{report.sequences.step.alerts}">Alerts</span>
+								<div class="lm2" th:if="${step.getAlerts().size() &gt; 0}">
+									<th:block th:each="alert: ${step.getAlerts()}">
+										<span th:class="'alert-' + ${alert.getRisk() + 'b'}">&nbsp;<span>
+												<a th:href="'#plugin-' + ${alert.getPluginId()}"
+												th:text="${alert.getName()}" href="#plugin-pluginId">Alert
```

Comment:
The `plugin-` prefix does not look right, the alerts have `th:href="'#' + ${alert.userObject.pluginId}"`.

Reply:
I probably assumed it was the same as plus....will check both